src/flashbar/flash.tsx

```diff
@@ -112,7 +113,9 @@ export const Flash = React.forwardRef(
             : ''
         )}
       >
-        <div className={clsx(styles['flash-icon'], styles['flash-text'])}>{icon}</div>
+        <div className={clsx(styles['flash-icon'], styles['flash-text'])} role="img" aria-label={statusIconAriaLabel}>
```

Comment:
The previously failing a11y tests actually showcase an important detail about this implementation. Because we're adding `role="img"` all the time now, all existing flash bars will now raise a11y errors because of missing labels. It would now be impossible to mark this icon as decorative.

Are we absolutely sure that we want to implement it this way? Or are there cases where the icon of a flash bar message can be safely considered decorative, especially for the "info" type?

We could render the `role` conditionally, based on whether the ARIA label was provided or not. What do you think?

Reply:
In my opinion, it is good to have the a11y error as a reminder to always add the arialabel. But I am also fine to make it only together with arialabel, only if there are cases icons can be considered decorative.

Reply:
I don't think we have any situations where the icon is decorative, so I too think we should let the errors occur as an incentive to fix this. We might just want to make sure we encourage users to not to start their messages with "Warning/Error/Info/Success"